tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml

```diff
@@ -60,7 +71,9 @@ prepare: |
 
   # modify and repack gadget snap to add a defaults section and use our own
   # prepare-device hook to use the fakedevicesvc
-  mkdir "$(nested_get_extra_snaps_path)"
+  EXTRA_SNAPS_PATH="$(nested_get_extra_snaps_path)"
+  mkdir -p "$EXTRA_SNAPS_PATH"
+  tests.cleanup defer rm -rf "$EXTRA_SNAPS_PATH"
```

Comment:
this may actually not be necessary at all as per @sergiocazzolato in https://github.com/snapcore/snapd/pull/10833

Reply:
Agree, this is automatically created and cleaned, so lines 74, 75 and 76 are not needed

Reply:
I removed this